docs/withdrawals/FAQs.md

```diff
@@ -15,7 +15,7 @@ Note that due to external provider limitations, the reference will be truncated
 There could be a number of reasons a withdrawal has been rejected:
 
 - The provided bank account or default bank account is not active.
-- There is already an active withdrawal.
+- The specified portfolio's mandate is not execution only and there is already an active withdrawal.
```

Comment:
Should we also mention full withdrawals, i.e., the withdrawal will be rejected if there is a full withdrawal, regardless of the mandate type?

Reply:
I felt like that was be covered by `There is insufficient balance to fulfil the withdrawal request.` bullet point

Reply:
Right, it makes sense